numba/cuda/tests/cudapy/test_math.py

```diff
@@ -575,8 +583,18 @@ def test_math_isnan(self):
     def test_math_isinf(self):
         self.unary_bool_template_float32(math_isinf, np.isinf)
         self.unary_bool_template_float64(math_isinf, np.isinf)
-        self.unary_bool_template_int32(math_isinf, np.isnan)
-        self.unary_bool_template_int64(math_isinf, np.isnan)
+        self.unary_bool_template_int32(math_isinf, np.isinf)
+        self.unary_bool_template_int64(math_isinf, np.isinf)
+
+    #------------------------------------------------------------------------------
+    # test_math_isfinite
+
+
+    def test_math_isfinite(self):
```

Comment:
The tests here only test with finite values - could you add some testing with -inf, +inf, and NaN as well please?